src/pages/HomePage/HomePage.tsx

```diff
@@ -34,6 +35,11 @@ const HomePage = () => {
       promise.abort()
```

Comment:
What is promise abort?

Reply:
It's Dat's code, and he did explain that in his commit.